server/depot.go

```diff
@@ -146,7 +145,7 @@ func (d fileDepot) writeDB(cn string, serial *big.Int, filename string, cert *x5
 
 	file, err := os.OpenFile(name, os.O_CREATE | os.O_RDWR | os.O_APPEND, dbPerm)
 	if err != nil {
-		fmt.Printf("Could not append to "+name+" : %q\n", err.Error())
+		fmt.Errorf("could not append to "+name+" : %q\n", err.Error())
```

Comment:
notice `fmt.Errorf` returns an error. so here just change it to 
`return fmt.Errorf("could not append to "+name+" : %q\n", err.Error())`